crates/query-engine/translation/tests/snapshots/tests__select_composite_column_complex.snap

```diff
@@ -43,20 +43,20 @@ FROM
                       (
                         SELECT
                           ("%0_make_person"."result").*
-                      ) AS "%3_nested_field_binding"
+                      ) AS "%3_make_person.result"
                       LEFT OUTER JOIN LATERAL (
                         SELECT
                           row_to_json("%4_nested_fields") AS "collected"
                         FROM
                           (
                             SELECT
-                              "%5_nested_field_binding"."address_line_1" AS "address_line_1",
-                              "%5_nested_field_binding"."address_line_2" AS "address_line_2"
+                              "%5_make_person.result.address"."address_line_1" AS "address_line_1",
+                              "%5_make_person.result.address"."address_line_2" AS "address_line_2"
```

Comment:
Urgh, this is difficult to read.

Information-wise it's much better than the arbitrary name, but the use of `.` suggests column projection happening in-line and it's only when you look closely that you see it's actually a quoted identifier.

Maybe we should change this scheme to e.g. `->` or `[<field>]` in a follow-up PR.